packages/e2e-tests/specs/editor/various/inserting-blocks.test.js

```diff
@@ -347,8 +347,10 @@ describe( 'Inserting blocks', () => {
 
 	it( 'shows block preview when hovering over block in inserter', async () => {
 		await openGlobalBlockInserter();
-		await page.waitForSelector( '.editor-block-list-item-paragraph' );
-		await page.focus( '.editor-block-list-item-paragraph' );
+		const paragraphButton = (
+			await page.$x( `//button//span[contains(text(), 'Paragraph')]` )
+		 )[ 0 ];
+		await paragraphButton.hover();
```

Comment:
Fixed one E2E test failure.
In this PR, `hover()` must be performed because the inserter is no longer visible when in `focus()`.